posts/2024-03-04_inside_the__phar.../inside_the__pharmaverse.qmd

```diff
@@ -0,0 +1,138 @@
+---
+title: "Inside the Pharmaverse"
+author:
+  - name: Michael Rimler
+description: "A short blog to help the Pharmaverse community understand how Pharmaverse is governed"
+date: "2024-03-04"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [community]
+# feel free to change the image
+image: "pharmaverse.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-03-04_inside_the__phar..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Greetings, Pharmaverse phriends!
+
+Now that the {pharmaverse} has been around for a few years and seems to be making an impact on clinical reporting (yay!), we wanted to take an opportunity to cast a light on some of the inner workings of Pharmaverse and its Council (us).
```

Comment:
Throughout the post, both {Pharmaverse} and Pharmaverse are used - suggest sticking to one?

Reply:
I used {Pharmaverse} once, intentionally, for style. Unless you see other instances, I think it's ok.